.requirements

```diff
@@ -6,7 +6,7 @@ OPENSSL=3.1.2
 PCRE=8.45
 LIBEXPAT=2.5.0
 
-LUA_KONG_NGINX_MODULE=4d19e8d19c6dbc07eba5cf6f5ebacad95266f928 # 0.6.0
+LUA_KONG_NGINX_MODULE=949131f4ccb841c9d9851b413be4a789f56ec7f3 # 0.7.0
```

Comment:
Since https://github.com/Kong/lua-kong-nginx-module/pull/61 fixed the test case about OpenResty 1.21.4.2, should we bump the version to 0.7.1?